### ISSUE TITLE
Close button outline

### ISSUE THREAD

**Issue by RavanH** [jsor/lity#181]
Hi, I'm not an expert but I think for accessibility reasons the style rule `outline:none` for .lity-close and .lity-close:focus should be removed.

**Comment by jsor** [comment]
👍 Feel free top open a PR

**Comment by RavanH** [comment]
Sure :)

Related question: there seem to be a lot of "double" css rules that appear both for .lity-close and .lity-close:hoverfocus/active/visited (except `text-decoration:none`) that could be removed. Or is there a special reason to keep them in both?

**Comment by RavanH** [comment]
Oh, hang on... this is not the case in current development. I was referring to 2.3.1. Never mind :)
